binding/HarfBuzzSharp.Shared/Buffer.cs

```diff
@@ -339,46 +271,39 @@ public void NormalizeGlyphs ()
 		public string SerializeGlyphs (Font font, SerializeFormat format, SerializeFlag flags) =>
 			SerializeGlyphs (0, -1, font, format, flags);
 
-		public string SerializeGlyphs (int start, int end, Font font, SerializeFormat format, SerializeFlag flags)
+		public unsafe string SerializeGlyphs (int start, int end, Font font, SerializeFormat format, SerializeFlag flags)
 		{
-			const int bufferSize = 128;
-
-			if (Length == 0) {
+			if (Length == 0)
 				throw new InvalidOperationException ("Buffer should not be empty.");
-			}
-
-			if (ContentType != ContentType.Glyphs) {
+			if (ContentType != ContentType.Glyphs)
 				throw new InvalidOperationException ("ContentType should be of type Glyphs.");
-			}
 
-			if (end == -1) {
+			if (end == -1)
 				end = Length;
-			}
 
-			var builder = new StringBuilder (128);
-			var buffer = Marshal.AllocCoTaskMem (bufferSize);
-			var currentPosition = start;
+			using (var buffer = MemoryPool<byte>.Shared.Rent ())
+			using (var pinned = buffer.Memory.Pin ()) {
```

Comment:
Looks like we can use the new memory pool types instead of allocating memory... Just need to now apply this to the rest of the repository, but this looks nice.